CONTRIBUTING.md

```diff
@@ -52,3 +53,18 @@ The best way to reach us with a question when contributing is to ask on:
 - The original github issue
 - The developer mailing list
 - Our Slack channel
+
+## How to Develop Odigos Locally
+The main steps involved when debugging Odigos locally are:
+
+- Use a Kind kubernetes cluster
+- Build custom images of Odigos and load them into Kind via:
```

Comment:
Perhaps add a link to odigos docs which describes the full steps to use
https://docs.odigos.io/intro